# HG changeset patch
# User Rik <rik@octave.org>
# Date 1359847853 28800
#      Sat Feb 02 15:30:53 2013 -0800
# Node ID 352ef2fb3ad1d7dfd4c8d9f33ca83636c6ef4460
# Parent  41471c02d51cc74d2cd1e0d28a4cff7d6857f526
Temporary fix for stack overflows with quadcc

quadcc.cc: Reduce number of allowed singularities on integration path from 200
to 50.

diff --git a/libinterp/corefcn/quadcc.cc b/libinterp/corefcn/quadcc.cc
--- a/libinterp/corefcn/quadcc.cc
+++ b/libinterp/corefcn/quadcc.cc
@@ -32,17 +32,17 @@ along with Octave; see the file COPYING.
 #include "error.h"
 #include "oct-obj.h"
 #include "utils.h"
 
 //#include "oct.h"
 //#include "defun.h"
 
 /* Define the size of the interval heap. */
-#define cquad_heapsize                  200
+#define cquad_heapsize                  50
 
 
 /* Data of a single interval */
 typedef struct
 {
   double a, b;
   double c[64];
   double fx[33];
